book.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './rental'
+
+class Book
+  def initalize(title, author)
+    @title = title
+    @author = author
+  end
+
+  attr_accessor :title, :author
```

Comment:
- [ ] Please move the association attributes to the top of the classes before the `initialize` method.
- [ ] Add another association for the rentals to check the association between the book and rental classes